### PR TITLE
Da thick blast update: blastcannon resprite.

### PR DESCRIPTION
**Pull request by Krysonism**
## About The Pull Request
This PR resprites the blastcannon sprite(item and inhand), as well as the blastwave projectile(inspired by Fikou's version).

It also slightly changes the description so it makes more sense.
bild
bild
blastwave_dm-export

## Why It's Good For The Game

The old blastcannon is one of the worst sprites in the game, and the old inhand is tiny.

A weapon that instagibs people should probably have a prominent inhand and now it does! Probably the biggest inhand of any item...

## Changelog

:cl:
imageadd: The blast cannon & blast wave projectile has new sprites.
/:cl:

### CODE REVIEW

**Comment by TemporalOroboros** [comment]
The sprites are awesome... they also look like the guys going to snap their wrist. Is there a way around that or does it just come with the perspective and sprite systems? Also, do you think you could do anything for the blast wave projectile? It is similarly maximum codersprite.

**Comment by Dorsisdwarf** [comment]
The sideways sprite might wanna be shifted back onto the wielder somewhat, since its so large and high contrast its definitely not gonna get lost with like a 4 pixel shunt, but this is an excellent sprite and a much needed change.

File: code/modules/projectiles/guns/special/blastcannon.dm [hunk 1]
```diff
@@ -84,7 +90,11 @@
 	return ..()
 
 /obj/item/gun/blastcannon/update_icon_state()
-	icon_state = "[bomb ? "loaded" : "empty"]_[base_icon_state]"
+	icon_state = "[base_icon_state]_[bomb ? "loaded" : "empty"]"
+	inhand_icon_state = icon_state
+	if(ismob(loc))
+		var/mob/cannoneer = loc
+		cannoneer.update_inv_hands()
```
Comment (Fikou) [inline code/modules/projectiles/guns/special/blastcannon.dm:97]: theres an element/component for this, called something like update icon update onmob

**Comment by Burgermancoding** [comment]
https://www.youtube.com/watch?v=8VysHWp6miM

File: code/modules/projectiles/guns/special/blastcannon.dm (hunk 1 above)
Comment (Krysonism) [reply code/modules/projectiles/guns/special/blastcannon.dm:97]: good catch!

**Comment by TemporalOroboros** [comment]
That's a very good sprite for the blastwave. :+1: 

Huh, looking at the old projectile sprite I think it was meant to fly sideways. No wonder it looked so jank. It was maximum codersprite and rotated 90 degrees at the same time.

**Comment by Coastfront** [comment]
This is the new penis cannon sprite.
This sprites head on look on someone holding it, looks likes a penis.
Penis cannon.
I approve of the cock artillery.

**Comment by SuperNovaa41** [comment]
Soul removal

**Comment by Bawhoppen** [comment]
> Soul removal

If the previous sprite was soulful, I wanna be soulless

**Review by Twaticus — APPROVED** [review]
comically huge

**Pull request merged**